CONTRIBUTING.md

```diff
@@ -0,0 +1,45 @@
+# Contributing Guidelines
+
+Hi! Nice to have you with us. This file will walk you through the steps you need to take if you want to contribute a word or some code to the project. Let’s go.
+
+Actually – before we start – a quick word of caution: Unfortunately the contribution process involves some technical knowledge. We aware that this poses a problem for most people and are discussing how a more inclusive contribution process could look like.
```

Comment:
No spaces around em dashes (the annoying grammatical hill I will die on!)

Reply:
<3